pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -211,6 +211,11 @@ type VolumeBindingArgs struct {
 	// Value must be non-negative integer. The value zero indicates no waiting.
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds int64
+
+	// Shape specifies the points defining the score function shape.
```

Comment:
> It would also be good to indicate that this field requires the alpha VolumeCapacityPriority feature gate to be enabled.
Is the `+featureGate=VolumeCapacityPriority` for this purpose?

Reply:
> Can you add more description to this field? What does each point represent and how does it impact scoring?

tried my best, but I'm not good at this, suggestions are welcome!

Reply:
Thanks, this is helpful.

Can you also describe how volume capacity maps to utilization? Is it based on #pvs, total capacity available to the node?

Reply:
> Can you also describe how volume capacity maps to utilization? Is it based on #pvs, total capacity available to the node?

done in https://github.com/kubernetes/kubernetes/pull/99594/commits/5d9baa9b2ac356b18b638dd6ba64723d6f12c3c7

the existing bound PVs on the node are excluded as they cannot be shared by other pods like cpu/memory.